textures/ReadMe.txt

```diff
@@ -1,2 +1,2 @@
-You will need to download bluemarble_16384.png from the project-atlantis website and save it to this folder. One way to do this is to use the following command:
-wget https://www.project-atlantis.com/wp-content/threejs-simulation/bluemarble_16384.png -outfile bluemarble_16384.png
+Run the command below to sync textures:
```

Comment:
Need to add a line here to be clear here about what directory the user needs to cd into before running this command.